test/playwright/README.md

```diff
@@ -0,0 +1,61 @@
+# Get Cmfive Working
+
+- Use this codespace: https://github.com/2pisoftware/codespace_dev_box/tree/BoilerplateCore_Modules_Tests_Debug

```

Comment:
This now outdated? Also, per chatter elsewhere around doc's, this is not a cmfive public repo/reference?

Reply:
will remove the "Get Cmfive Working" part, was for Dane to follow while trying to get the newer codespace to be playwright friendly